### PR TITLE
Bug 1115130 - Make building for raptor turn on user timing logging preference

### PR DESCRIPTION
**Pull request by qdot**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[qdot Kyle Machulis (qdot)](https://github.com/qdot) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=dc47659cc3a3)

File: build/preferences.js [hunk 1]
```diff
@@ -174,6 +174,10 @@ PreferencesBuilder.prototype.preparePref = function() {
   if (this.config.DEVICE_DEBUG === '1') {
     this.setDeviceDebugPref();
   }
+  // If we're turning RAPTOR on, also turn on user timing output to logcat
+  if (this.config.RAPTOR == '1') {
```
Comment (eliperelman) [inline build/preferences.js]: nit: `===` for consistency

**Comment by try-server-hook** [comment]
[qdot Kyle Machulis (qdot)](https://github.com/qdot) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=f35b1290f15d)

**Comment by qdot** [comment]
nits addressed, pref name updated. Merging.

**Pull request merged**